include/openthread-config-android-version.h.in

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2021, The OpenThread Authors.
```

Comment:
nit: can we move this file to something like `etc/android`? Similar to what we did with `etc/cmake/openthread-config-generic.h.in`?

Reply:
Updated